dashbuilder/dashbuilder-shared/dashbuilder-js/packages/component-api/src/__tests__/api.test.ts

```diff
@@ -13,9 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import * as ComponentAPI from "../controller/api";
-import * as Bus from "../controller/ComponentBus";
+import { Bus } from "../controller";
+import * as ComponentAPI from "../controller";
```

Comment:
Hello Tiago,

I do agree with you. This is what the API should provide to component developers :

1) Model Objects that represents what is sent in the wire:

* DataSet
* FunctionRequest
* FilterRequest

and others objects

2) Access to ComponentController, which is how users will communicate with DB. In practice only "getComponentController" would be required so users can set their callback methods and call functions/fire configuration issues alert and more. To expose this function I simply exported it from controller, but I moved api.ts to the root dir, it will avoid confusion. 

When importing it I usually call it ComponentAPI and in the component itself users should not see controller package, I hope only the getComponentController and the model objects would be accessible!

![image](https://user-images.githubusercontent.com/359820/101513127-9388fb80-395a-11eb-89c4-8d67722e3985.png)